update: Changelog for Cycle 2412 (#140)
vatnz-dev/sops: docs/changelog/2024/2412.md

---
  title: Cycle 2412 (28 Nov)
---

--8<-- "includes/abbreviations.md"

> Cycle 12 is always a major update for NZZC, with this year being no different. Cycle 2412 implements Auckland DMAPs, ensuring further separation between departures and missed approach paths. There are also a plethora of changes to IFR waypoints.

Reviewer: ```suggestion
Cycle 12 is always a major update for NZZC, with this year being no different. Cycle 2412 implements Auckland DMAPS, ensuring further separation between departures and missed approach paths. There are also a plethora of changes to IFR waypoints.
```

Reviewer: Feel like I'm teaching a grandmother how to suck eggs but I note that in Airways seem to refer to it as DMAPS (capital S). Which is good because probably would be good to stick DMAPS into the abbreviations to help spell out what it stands for :)

Author: Amended, and added to the abbreviations.

## SOP Changes

> ### TBC (Pending Mikey's Changes)

Reviewer: @VATNZ-MR Were you wanting to add some stuff into this section?

Reviewer 2: Done!

-----

## Data Changes

- The following major waypoints have been disestablished, and replaced:
    - `LAKES` replaced by `ORIDI`
    - `KARRL` replaced by `IGBAT`
    - `GOOSE` replaced by `IDGIV`

### Aerodrome

#### NZAA: Auckland

There are significant changes to Auckland, with the implementation of the DMAPs, resulting in procedure and airspace changes. Additionally, the CTR/C boundary has been amended, and Visual Reporting Points replaced.

- The lateral bounds of the Auckland CTR/C have been amended:
    - North Sector now split into three subsectors: `North West`, `North` and `North East`.
    - South Sector now split into two subsectors: `South West` and `South East`.
    - Additional pieces of CTA/C airspace added.
- All VRPs within the NZAA CTA/C have been removed, and new ones added.

> - `MANGERE BRIDGE` VFR arrival and departure have been replaced with the `AMBURY` arrival and departure.

Reviewer: ```suggestion
- `MANGERE BRIDGE` VFR arrival and departure has been replaced with the `AMBURY` arrival and departure.
```

#### NZWP: Whenuapai 

- The following SID has been amended and resequenced: `BAYES 2R`.
    - `LAKES` transition replaced by `ORIDI`.
- The following SIDs have been added: `NZWP 1A`, `NZWP 1B`.

#### NZWB: Woodbourne

- The following SIDs have been amended and resequenced: `VINES 5`, `VALLEY 4`.
    - `VINES 5` departure routing amended.
- The following SIDs have been added: `BEAVER 1`.

-----

### Standard Routes

- The following STD RTEs have been amended and resequenced: `AANP2`, `AAPM3`, `AAPP1`, `AAWN2`, `CHWN7`, `WRWN1`, `WPOH1`, `WPWN2`.
- The following STD RTEs have been added: `ARHN1`
- All Dunedin STD RTEs have had their remarks amended to clarify the `FL260` or higher requirement, with the `FL250` hold-down now reflected in DN TWR SOPs.
- Further standardised remark values.
- Routes have been updated in the Standard Route Tool and [public route repository](https://github.com/vatnz-dev/std-rte-public){ target=blank }.

### VRPs for LittleNavMap

The VRPs for LittleNavMap package has been updated, and [may be downloaded here](https://github.com/vatnz-dev/vrpsForLittleNavMap/releases/tag/2412){ target=blank }.

### Other Dataset Changes

> - The Hamilton Delivery position (HN_DEL) has been removed.

Reviewer: This one is referenced both here and in the data changes for NZHN. Guessing we should probably go with one or the other?

Author: I think we should leave it in the Hamilton section - as it's a AIP-directed change, as opposed to a VATNZ-directed change.

> - The vatSys profile has been renamed `VTANZ Domestic (NZZC)` to align with the Combined Oceanic dataset.

Reviewer: ```suggestion
- The vatSys profile has been renamed `VATNZ Domestic (NZZC)` to align with the Combined Oceanic dataset.
```
Or at least I hope it's not named VTANZ in the other repo 😅

---
  title: Cycle 2412 (28 Nov)
---

--8<-- "includes/abbreviations.md"

Cycle 12 is always a major update for NZZC, with this year being no different. Cycle 2412 implements Auckland DMAPS, ensuring further separation between departures and missed approach paths. There are also a plethora of changes to IFR waypoints.

## SOP Changes

### Content

The following content has been added:

- Nelson Aerodrome
- Queenstown TMA
- Client Guides for vatSys and Euroscope

### Aerodromes

> #### NZAA: Auckland

Reviewer: Also probably worth calling out the main waypoint changes as a pointer for people to take a closer look?

Reviewer 2: Already highlighted under "Data Changes"

## SOP Changes

### Content

The following content has been added:

- Nelson Aerodrome
- Queenstown TMA
- Client Guides for vatSys and Euroscope

### Aerodromes

#### NZAA: Auckland

- VFR Procedures have been updated

>     - VFR Aircraft can now expect to be given an `AMBURY` Arrival and to join via "Little Creek".

Reviewer: ```suggestion
    - VFR Aircraft can now expect to be given an `AMBURY` Departure and Arrival.
```
Suggested to add the change to departures too and to remove the join instructions as that's more of a detail for the SOPs itself
